DTS-STN/Service-Canada-Labs
# Benefit estimator page

# Description

[create "OAS Benefits Estimator" overview page](https://dev.azure.com/VP-BD/DECD/_sprints/taskboard/Service%20Canada%20Labs/DECD/Service%20Canada%20Labs/SCL%20Sprint%2010?workitem=102619)

Created the Benefits Estimator overview page. Page data fetched from AEM. Unit testing created. 

## Acceptance Criteria

The new page should look like the design on [Figma](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?node-id=11261%3A29128)

## Test Instructions

1. vists /projects/oas-benefits-estimator
2. check the content for both English and French
3. Make sure it matches the design

## Checklist
